dist/tools/usb-cdc-ecm/start_network.sh

```diff
@@ -97,7 +97,7 @@ fi
 
 if [ -z "${PORT}" ]; then
     echo "Network enabled over CDC-ECM"
-    read -n 1 -s -p "Press any key to stop"
+    read "Press Return to stop"
```

Comment:
```suggestion
    read -p "Press Return to stop"
```

For busybox ash and bash the `-p` is needed. For `zsh`, it breaks with `-p`.

Or maybe better:

```
    echo "Network enabled over CDC-ECM"
    echo "Press Return to stop"
    read
```

The last one actually is the only one consistent with `man 1 read`, which documents `-r` as the only supported flag.

Reply:
(I just realized you already suggested using a plain `read`. +1 for that.)